x/ratelimit/client/cli/cli_test.go

```diff
@@ -0,0 +1,318 @@
+package cli_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+
+	strideclitestutil "github.com/Stride-Labs/stride/v4/testutil/cli"
+
+	"github.com/Stride-Labs/stride/v4/testutil/network"
+
+	"github.com/stretchr/testify/suite"
+
+	tmcli "github.com/tendermint/tendermint/libs/cli"
+
+	"github.com/Stride-Labs/stride/v4/x/ratelimit/client/cli"
+
+	"github.com/Stride-Labs/stride/v4/app"
+	"github.com/Stride-Labs/stride/v4/x/ratelimit/types"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	cfg     network.Config
+	network *network.Network
+}
+
+func (s *IntegrationTestSuite) SetupSuite() {
+	s.T().Log("setting up integration test suite")
+
+	s.cfg = network.DefaultConfig()
+
+	genState := app.ModuleBasics.DefaultGenesis(s.cfg.Codec)
+	ratelimitGenState := types.DefaultGenesis()
+	ratelimitGenStateBz := s.cfg.Codec.MustMarshalJSON(ratelimitGenState)
+	genState[types.ModuleName] = ratelimitGenStateBz
+
+	s.cfg.GenesisState = genState
+	s.network = network.New(s.T(), s.cfg)
+
+	_, err := s.network.WaitForHeight(1)
+	s.Require().NoError(err)
+}
+
+func (s *IntegrationTestSuite) TestCmdTxAddRateLimit() {
```

Comment:
Shouldn't these tests fail since the channel value is zero at the time that they're created?

Reply:
No, when the testnet for cli tests starts running, some bond_denom tokens are minted initially.
Please check `testutil/network/network.go` => `DefaultConfig` function.

Reply:
got it, thanks for the clarification!